src/configurations/DRGBTConfig.cpp

```diff
@@ -4,15 +4,16 @@
 
 #include "DRGBTConfig.h"
 
-unsigned long DRGBTConfig::MAX_NUM_ITER             = 10e6;
-int DRGBTConfig::MAX_ITER_TIME                      = 100;
-int DRGBTConfig::MAX_PLANNING_TIME                  = 60e3;
+unsigned long DRGBTConfig::MAX_NUM_ITER             = 1e9;
+float DRGBTConfig::MAX_ITER_TIME                    = 0.050;
+float DRGBTConfig::MAX_PLANNING_TIME                = 60;
 int DRGBTConfig::INIT_HORIZON_SIZE                  = 10;
```

Comment:
Change obviously positive integer types to size_t

Reply:
OK, tnx.